examples/viewer.html

```diff
@@ -387,6 +423,23 @@
     }
 
 
+    function parseUrl(urlString) {
+      var a = document.createElement('a');
```

Comment:
I'd rather get to a point where we can refactor gulp to compile such things with ease via es6.